core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -1401,17 +1403,16 @@
     <xs:attribute name="size" type="xs:long" default="-1">
       <xs:annotation>
         <xs:documentation>
-          Defines the size of the data container as a long. Eviction occurs
-          when the number of entries exceeds the size.
+          Eviction occurs when the number of entries exceeds the size.
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
     <xs:attribute name="strategy" type="tns:eviction-strategy">
       <xs:annotation>
         <xs:documentation>
-          Specifies a strategy for evicting cache entries. Eviction always
-          takes place when you define the size of the data container. If you
-          specify a value for size, then you should configure a strategy.
+          Eviction always takes place when you define the size of the data
+          container. If you specify a value for size, then you should configure
```

Comment:
I wouldn't say they should. The default of REMOVE is probably just fine for the vast majority of use cases.